cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,80 @@
+import { generateUser } from '../support/commands';
+
 /// <reference types='cypress' />
 
+const user = generateUser();
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should be elements of pagination is present', () => {
+    cy.get('.-pageInfo').should('exist');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-pageJump > input').should('exist');
+    cy.get('.-totalPages').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+  });
+
+  it('should add new user', () => {
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.userEmail);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+
+    cy.get('#submit').click();
+
+    cy.get('.rt-table').should('contain', user.firstName, user.lastName,
+      user.email, user.age, user.salary, user.department);
```

Comment:
There is a mistake in the assertion. The method 'should' with 'contain' only accepts two arguments: the selector and the expected content. You should verify each user attribute separately or use a different assertion method.